### ISSUE TITLE
Error on config import

### ISSUE THREAD

**Issue by lordal** [canariecaf/idp-installer-CAF#93]
When importing an existing config servlet container can be changed.
appserv='blargh' in config import will cause an failure.

**Comment by canariecaf** [comment]
fair enough, it's a possible issue.  The 'defensive' code response to this is to enforce only valid configurations in the javascript and then alert the user. 
Alternatively and MORE work is to encode a controlled vocabulary into the installer code base and halt before running during pre-flight check.

We may want to consider doing both to protect ourselves.  While I think this is a low probability event, the removal of tomcat in favour of jetty does prove it could happen programatically.

I've placed this as a future enhancement into the .next stream
